Source/App.js

```diff
@@ -1 +1,417 @@
+            duration: 1.0
+        });
+    });
+
+    //Set up clock and timeline.
+    viewer.clock.shouldAnimate = true;
+    viewer.clock.clockStep = Cesium.ClockStep.TICK_DEPENDENT;
+    viewer.clock.clockRange = Cesium.ClockRange.LOOP_STOP; // loop at the end
+    viewer.clock.multiplier = 60; // sets a speedup
+    viewer.clock.startTime = Cesium.JulianDate.fromIso8601("2016-08-01T04:00:00Z");
+    viewer.clock.stopTime = Cesium.JulianDate.fromIso8601("2016-08-02T04:00:00Z");
+    viewer.clock.currentTime = Cesium.JulianDate.fromIso8601("2016-08-01T16:00:00Z");
+    viewer.timeline.zoomTo(viewer.clock.startTime, viewer.clock.stopTime); // set visible range
+
+    //Enable lighting based on sun/moon positions
+    viewer.scene.globe.enableLighting = true;
+
+    //////////////////////////////////////////////////////////////////////////
+    // Loading Imagery
+    //////////////////////////////////////////////////////////////////////////
+
+    Cesium.BingMapsApi.defaultKey = 'AsarFiDvISunWhi137V7l5Bu80baB73npU98oTyjqKOb7NbrkiuBPZfDxgXTrGtQ'; // For use on cesiumjs.org only
+
+    // Add Bing imagery
+    viewer.imageryLayers.addImageryProvider(new Cesium.BingMapsImageryProvider({
+        url : 'https://dev.virtualearth.net',
+        mapStyle: Cesium.BingMapsStyle.AERIAL // Can also use Cesium.BingMapsStyle.ROADS
+    }));
+
+    //////////////////////////////////////////////////////////////////////////
+    // Loading Terrain
+    //////////////////////////////////////////////////////////////////////////
+
+    //Load STK World Terrain
+    viewer.terrainProvider = new Cesium.CesiumTerrainProvider({
+        url : 'https://assets.agi.com/stk-terrain/world',
+        requestWaterMask : true, // required for water effects
+        requestVertexNormals : true // required for terrain lighting
+    });
+    //Enable depth testing so things behind the terrain disappear.
+    viewer.scene.globe.depthTestAgainstTerrain = true;
+
+    //////////////////////////////////////////////////////////////////////////
+    // Loading Data
+    //////////////////////////////////////////////////////////////////////////
+
+    var geojsonOptions = {
+        markerSize : 40,
+        markerSymbol : '!',
+        clampToGround : true
+    };
+    // Load points of interest from a GeoJson
+    var pointsPromise = Cesium.GeoJsonDataSource.load('./Source/SampleData/pointsOfInterest.geojson', geojsonOptions);
+
+
+    var kmlOptions = {
+        camera : viewer.scene.camera,
+        canvas : viewer.scene.canvas
+    };
+    // Load neighborhood boundaries from KML file
+    var neighborhoodsPromise = Cesium.KmlDataSource.load('./Source/SampleData/neighborhoods.kml', kmlOptions);
+
+    //////////////////////////////////////////////////////////////////////////
+    // Styling Data
+    //////////////////////////////////////////////////////////////////////////
+
+    var distanceDisplayCondition = new Cesium.DistanceDisplayCondition(10.0, 10000.0);
```

Comment:
I think the far value could be a little further out.  Especially for the `!` billboards